app.js

```diff
@@ -1,166 +0,0 @@
-document.addEventListener("DOMContentLoaded", () => {
-  let comments = [];
-  // Target DOM Elements
-  const comment_box = document.querySelector("#comment_box");
-  const comment_wrapper = document.querySelector("#comment-listing");
-  const nestedCommentForm = document.querySelector(
-    "#comment_list_inner .comment-area"
-  );
-
-  //initialize loacl stroge if not
-  if (localStorage.getItem("comments") === null)
-    localStorage.setItem("comments", JSON.stringify(comments));
-
-  //Add comment object in localStorge
-  const addComment = (comment) => {
-    let all_comments = getItemFromLocalStorage();
-    all_comments.push(comment);
-    setItemInLocalStorage(all_comments);
-  };
-
-  //update local stroge with new data
-  const setItemInLocalStorage = (newData) => {
-    localStorage.setItem("comments", JSON.stringify(newData));
-    renderComments();
-  };
-
-  //Add reply on the same index
-  const addReply = (reply, index) => {
-    let all_comments = getItemFromLocalStorage();
-    all_comments.find((element, i) => {
-      console.log(element);
```

Comment:
console logs should be something that can be omitted from the code, only add when you are debugging